src/lib/core/channel-value/channel-value.component.ts

```diff
@@ -53,7 +54,14 @@ export class ChannelValueComponent implements OnChanges {
     /** The text to display for the units (light text) */
     @Input() units: string;
     /** The text to display for the value (bold text)*/
-    @Input() value: string | number;
+    @Input('value')
+    get value(): string | number {
+        return this._value;
+    }
+    set value(newValue: string | number) {
+        this._updateRenderedValue(newValue);
+    }
+    private _value: string | number;
```

Comment:
We should be able to achieve this fix without the need for a getter and setter on that input.